### PR TITLE
[platform]: Add plugins for ingrasys

### PR DESCRIPTION
**Pull request by kaiyu22**
* Adding two utilities: eeprom.py and sfputil.py

Signed-off-by: Sam Yang <yang.kaiyu@gmail.com>

### CODE REVIEW

**Comment by lguohan** [comment]
@jleveque , can you review?

File: device/ingrasys/x86_64-ingrasys_s9100-r0/plugins/sfputil.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        30: 41,
+        31: 40
+    }
+
+    _qsfp_ports = range(0, ports_in_block + 1)
+
+    def __init__(self, port_num):
+        # Override port_to_eeprom_mapping for class initialization
+        eeprom_path = '/sys/class/i2c-adapter/i2c-{0}/{0}-0050/eeprom'
+        for x in range(0, self.port_end + 1):
```
Comment (jleveque) [inline device/ingrasys/x86_64-ingrasys_s9100-r0/plugins/sfputil.py]: I've noticed in a few versions of this file that `port_start` is defined but not used. For clarity, should use:

`for x in range(self.port_start, self.port_end + 1):`

Comment (kaiyu22) [inline device/ingrasys/x86_64-ingrasys_s9100-r0/plugins/sfputil.py]: Sure, no problem.

**Comment by kaiyu22** [comment]
@jleveque: I have updated the sfputil.py, please review again.

**Pull request merged**